presentation/src/main/java/com/knocklock/presentation/lockscreen/service/LockScreenNotificationListener.kt

```diff
@@ -79,38 +80,25 @@ class LockScreenNotificationListener :
         )
     }
 
-    @Inject lateinit var insertNotificationUseCase: InsertNotificationUseCase
-
     override fun onNotificationPosted(sbn: StatusBarNotification?) {
         super.onNotificationPosted(sbn)
 
         val packageName = sbn?.packageName
         if (sbn != null && !TextUtils.isEmpty(packageName)) {
-            val notification: Notification = sbn.notification
-            val extras = notification.extras
-            val title = extras.getString(Notification.EXTRA_TITLE).toString()
-            val text = extras.getString(Notification.EXTRA_TEXT).toString()
-            val subText = extras.getString(Notification.EXTRA_SUB_TEXT).toString()
-            val smallIcon = notification.smallIcon
-            val largeIcon = notification.getLargeIcon()
-
-            if (title.isNotBlank() && text.isNotBlank()) {
-                scope.launch {
-                    insertNotificationUseCase(
-                        NotificationDomainModel(
-                            id = 0,
-                            title = title,
-                            subText = subText,
-                            text = text
-                        )
-                    )
-                }
+            notificationScope.launch {
```

Comment:
여기서 코루틴 Job이 최초로 생성되는 것 같은데
이후 작업에서 Main Thread에서 실행되어야 하기 때문에 수정이 필요할 듯 하네요. (근본적인 이유는 모르겠네요 ㅠ)
```suggestion
            notificationScope.launch(Dispatchers.Main) {
```

Reply:
@jaeryo2357 [8de8579](https://github.com/Knock-Lock/Knock-Lock/pull/75/commits/8de8579639f450f0078fa7658edef67851e734c6)

View가 초기화 되는 시점을 MainThread로 변경했습니당